crates/ef-testing/src/evm_sequencer/evm_state/v1.rs

```diff
@@ -259,8 +259,8 @@ pub(crate) fn compute_storage_base_address(
     StorageKey(PatriciaKey::try_from(key_floored).unwrap()) // infallible
 }
 
-pub(crate) fn offset_storage_base_address(base_address: StorageKey, offset: i64) -> StorageKey {
-    let base_address = stark_felt_to_felt(*base_address.0.key()) + Felt252::from(offset);
+pub(crate) fn offset_storage_key(key: StorageKey, offset: i64) -> StorageKey {
```

Comment:
feels weird that this is semantically an addition, and that we somehow cannot offset directly the storageKey

Reply:
I know :/ the API on `StorageKey` is pretty bad, they should just implement something like `Add`, would make it a lot easier to offset, maybe PR on starknet api?